apps/submission/models.py

```diff
@@ -61,3 +62,19 @@ def archive_upload_to(instance, filename):
     )
 
     imported = models.BooleanField(default=False)
+
+    @property
+    def is_failed(self):
+        """Check if process has failed tasks"""
+
+        if self.task_set.filter(status=STATUS.ERROR).count() > 0:
```

Comment:
So we do not set any `status` to the main object when one of the tasks errored? Like it is done for `DONE` after.

Reply:
Unfortunately no. I had the same expectation, but, after reading viewflow's code again and again, it appears that it has been designed this way, _i.e._ a task error is never propagated to its process. Odd.